hack/check-pr.sh

```diff
@@ -21,11 +21,20 @@ set -o nounset
 set -o errexit
 set -o pipefail
 
+cd `git rev-parse --show-toplevel`
+
 dirs=()
 tests=()
 ref="${1:-HEAD}"
 echo -n "Packages changed since $ref: "
-for d in $(git diff --name-only "$ref" | xargs -n 1 dirname | sort -u); do
+
+changed=$(git diff --name-only "$ref")
+if [[ ${#changed} == 0 ]]; then
```

Comment:
`[[ -n "$changed" ]]`

http://tldp.org/LDP/Bash-Beginners-Guide/html/sect_07_01.html

Reply:
I can't seem to get that to work (?) so I'll keep it the way it is for now

Reply:
`-z`

Reply:
`-n` non-zero `-z` zero